Simulator/Layouts/Layout.cpp

```diff
@@ -52,30 +45,11 @@ Layout::Layout() : numEndogenouslyActiveNeurons_(0)
 /// Destructor
 Layout::~Layout()
 {
```

Comment:
Do we leave destructors empty, or do we delete their implementations to rely on the default behavior (which is the same thing...)?

Reply:
I was in a dilemma about this too. I will go ahead and remove the empty destructor implementation as it is cleaner and mark the destructor declaration =default to maintain readability.